specification/databoxedge/resource-manager/Microsoft.DataBoxEdge/stable/2020-09-01/databoxedge.json

```diff
@@ -0,0 +1,8511 @@
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "monitoringMetricConfiguration",
+            "in": "body",
+            "description": "The metric configuration.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/MonitoringMetricConfiguration"
+            }
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted the request to create or update the metrics configuration."
+          },
+          "200": {
+            "description": "Successfully created or updated the metrics configuration.",
+            "schema": {
+              "$ref": "#/definitions/MonitoringMetricConfiguration"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "PutMonitoringConfig": {
+            "$ref": "./examples/PutMonitoringConfig.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "MonitoringConfig"
+        ],
+        "summary": "deletes a new metric configuration for a role.",
+        "operationId": "MonitoringConfig_Delete",
+        "parameters": [
+          {
+            "name": "deviceName",
+            "in": "path",
+            "description": "The device name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "roleName",
+            "in": "path",
+            "description": "The role name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully deleted the metrics configuration."
+          },
+          "202": {
+            "description": "Accepted the request to delete the metrics configuration."
+          },
+          "204": {
+            "description": "The metrics configuration is already deleted."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
```

Comment:
Doesn't the API return 'Location' header to track async DELETE?
https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#delete-resource-asynchronously

Reply:
Yes it returns location header to track the requests